vnext/Microsoft.ReactNative/Utils/BatchingEventEmitter.cpp

```diff
@@ -40,20 +36,18 @@ void BatchingEventEmitter::EmitJSEvent(
 }
 
 void BatchingEventEmitter::EmitCoalescingJSEvent(
-    int64_t tag,
-    winrt::hstring &&eventName,
-    JSValue &&eventObject) noexcept {
-  return EmitCoalescingJSEvent(L"receiveEvent", tag, std::move(eventName), std::move(eventObject));
+    winrt::hstring &&coalescingKey,
+    JSValueArgWriter &&paramsWriter) noexcept {
```

Comment:
I think this is consistent with other event APIs, and it's generic enough for special cased `touchMove`, but there are some downsides moving away from named parameters. Untyped args means you really need to look at usage to understand what you're supposed to pass for this. Wonder if there is a way to reconcile the two.